Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/BroadcasterSettings.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    /// <summary>
+    /// Settings used by a <see cref="Broadcaster"/>
+    /// </summary>
+    public class BroadcasterSettings : Singleton<BroadcasterSettings>
+    {
+        /// <summary>
+        /// Determines whether or not all GameObjects are synchronized or only those with a GameObjectHierarchyBroadcaster are synchronized.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Determines whether or not all GameObjects are synchronized or only those with a GameObjectHierarchyBroadcaster are synchronized.")]
+        private bool automaticallySynchronizeAllGameObjects = false;
+
+        /// <summary>
+        /// Determines whether or not all GameObjects are synchronized or only those with a GameObjectHierarchyBroadcaster are synchronized.
```

Comment:
>synchronized [](start = 58, length = 12)

In general it might be worth searching for the word 'synchronized' and evaluating whether or not it should be replaced with 'broadcast'.